src/components/molecules/NavSearchBar/NavSearchList.scss

```diff
@@ -0,0 +1,10 @@
+@import "scss/constants.scss";
+@import "./navSearchBarConstants.scss";
+
+.NavSearchList {
+  &__list {
+    @include scrollbar;
+    overflow-y: auto;
+    max-height: $search-results-height--max;
+  }
```

Comment:
Moved from `./NavSearchBar.scss` and renamed to keep in line with the component that contains the style